content/millennium/r4/scheduling/appointment.md

```diff
@@ -249,6 +249,17 @@ Create a new Appointment.
 
     POST /Appointment
 
+_Implementation Notes_
+
+* The modifier elements [implicitRules] and [modifierExtension] are not supported and will be rejected if present.
+* `Appointment.status` must be set to `accepted`.
+* `Appointment.slot` must be a list containing a single reference to the Slot in which this appointment is being booked.
+* `Appointment.slot[0].reference` specifies an availability in the Scheduling system, which indicates details such as practitioner, location, and time.
+* `Appointment.participant` must have exactly one participant.
+* `Appointment.participant.status` must be set to `accepted`.
+* `Appointment.participant.type` must not be set.
+* `Appointment.comment` must be a string.
```

Comment:
Looks like we do not have any business rules around comments for R4.

Reply:
deleted comments here: https://github.com/cerner/fhir.cerner.com/pull/311/commits/9dc3c856e57a3a3caffe0ea6727b0e4c272332e1